test/test-helper.el

```diff
@@ -15,7 +17,8 @@
               indent-tabs-mode nil)
 
 ;; Load the elixir-mode under test
-(require 'elixir-mode)
+;; might be alchemist
+;; (require 'elixir-mode)
```

Comment:
On 17.03.2015 21:26, Matt DeBoard wrote:
>> @@ -15,7 +17,8 @@
>>                 indent-tabs-mode nil)
>>
>>   ;; Load the elixir-mode under test
>> -(require 'elixir-mode)
>> +;; might be alchemist
>> +;; (require 'elixir-mode)
>
> Can you explain why this gets commented out here instead of detecting whether or not it should be run?
>

Thought at loading it from within a macro. Restored for the moment.gis